docs/operations/rule-configuration.md

```diff
@@ -167,7 +167,7 @@ Set the following properties:
   - the segment interval starts any time after the rule interval starts.
 
   You can use this property to load segments with future start and end dates, where "future" is relative to the time when the Coordinator evaluates data against the rule. Defaults to `true`.
-- `tieredReplicants`: a map of tier names to the number of segment replicas for that tier.
+- `tieredReplicants`: a map of tier names to the number of segment replicas for that tier. If you set the replicants for a period to 0 on all tiers, you can still [query the data from deep storage](../querying/query-from-deep-storage.md).
```

Comment:
Another way to do query the data from deep storage is to 
set `tieredReplicants` empty and set `useDefaultTierForNull` to false. I think we should push users to this way in the docs. 
cc @adarshsanjeev